README.md

```diff
@@ -63,6 +63,12 @@ LayerLens marks inverted dependencies (dependencies that create circles) with '!
 
 Also you can add command `dart run layerlens --fail-on-cycles` to the repo's pre-submit bots.
 
+## Build filters
+
+If you want to generate the `DEPENDENCIES.md` only for a specific folders, you can use `--build-filter` option and you should use [glob](https://pub.dev/packages/glob) syntax. For example, to generate the diagram only for the root `lib/` folder, you run following `dart run layerlens --build-filter "lib"`.
+
+You can specify multiple build filters . The mechanism is inspired by `--build-filter` in Dart's [`build_runner`](https://github.com/dart-lang/build/blob/master/docs/partial_builds.md). For example, to run the layerlens for root `lib/` and it's subfolder `lib/subfolder1` run `layerlens --build-filter "lib" --build-filter "lib/subfolder1"`
```

Comment:
Thanks.
Can you also add example how to generate diagrams for entire subtrre of one of subfolders?

Reply:
Good point, I've updated the readme and added this case to the tests as well